File: Fundamentals of C programming/step06/6.3/6.3.02.c
/*

Даны три числа. Подсчитать количество положительных среди этих чисел.

Входные данные:
Три целых числа a,b,ca,b,c.

Выходные данные:
Одно целое число -- количество положительных чисел, среди чисел a,b,ca,b,c.

Sample Input:

79 -18 88

Sample Output:

2

*/

#include <stdio.h>

int main() {
  int a,b,c,count=0;
  scanf("%d %d %d",&a,&b,&c);
  
  if (a > 2) {
    count = count+1;
 } if (b > 2) {
   count = count+1;
 } if (c > 2) {
   count = count+1;
 }

 printf("%d\n",count);

  return 0;
}

File: Diving into JavaScript: For Beginners/step06/6.03/6.3.02.js
/*

Перед вами программа, которая должна запрашивать баланс банковского счета у пользователя и выводить сообщение о балансе в заявленном формате. 
Вставьте вместо многоточия объявление метода для объекта.

Sample Input 1:

800

Sample Output 1:

Ваш текущий баланс: 800 долларов

Sample Input 2:

5

Sample Output 2:

Ваш текущий баланс: 5 долларов

*/

const balance = Number(prompt())        
let bankAccount = {
  balance,
  checkBalance() {
      console.log('Ваш текущий баланс:',this.balance,'долларов')
  }
};
bankAccount.checkBalance();
